kolibri/core/assets/src/state/actions.js

```diff
@@ -428,39 +428,53 @@ function saveLogs(store) {
   }
 }
 
+function fetchPoints(store) {
+  if (!getters.isSuperuser(store.state) && getters.isUserLoggedIn(store.state)) {
+    const userProgressModel = UserProgressResource.getModel(getters.currentUserId(store.state));
+    userProgressModel.fetch().then(progress => {
+      store.dispatch('SET_TOTAL_PROGRESS', progress.progress);
+    });
+  }
+}
+
 /**
- * Update the progress percentage
- * To be called periodically by content renderers on interval or on pause
- * Must be called after initContentSession
- * @param {float} progressPercent
- * @param {boolean} forceSave
+ * Helper function to handle common functionality between updateProgress and updateExerciseProgress
+ * @param  {VuexStore} store        The currently active Vuex store
+ * @param  {Number} sessionProgress The progress made in this session
+ * @param  {[type]} summaryProgress The progress made on this content overall
+ * @param {boolean} forceSave       Force saving of logs?
  */
-function updateProgress(store, progressPercent, forceSave = false) {
+function _updateProgress(store, sessionProgress, summaryProgress, forceSave = false) {
```

Comment:
Would be helpful to give the helper function a more semantic name, less associated with `updateProgress`